cnf-certification-test/tnf_config.yml

```diff
@@ -1,21 +1,23 @@
 targetNameSpaces:
   - name: tnf
+  - name: default # Added for testing cr-scale-operator, as it is installed in default namespace
```

Comment:
defulat is not a good ns 
"description": "Tests that all CNF's resources (PUTs and CRs) belong to valid namespaces. A valid namespace meets\nthe following conditions: (1) It was declared in the yaml config file under the targetNameSpaces\ntag. (2) It does not have any of the following prefixes: default, openshift-, istio- and aspenmesh-"
maybe change the installation ns?

Reply:
Installed in tnf